src/Dan.Plugin.Arbt/Models/Unit.cs

```diff
@@ -217,4 +218,61 @@ public class Unit
     }
 
 
+    public class BilpleieResponse
```

Comment:
Nå er alle forretningsmodellene for de tre registrene lagt inn sammen med Unit-modellene fra enhetsregisteret. Foreslår du flytter disse ut til egne filer.